tokio-test/src/io.rs

```diff
@@ -1,4 +1,5 @@
 #![cfg(not(loom))]
+#![allow(dead_code)]
```

Comment:
What is dead code in here? We probably should remove the dead code.

Reply:
rx is never used......and its the same issue in the current version RA seems to complain.